scripts/setup_gemdaq.sh

```diff
@@ -329,6 +329,10 @@ then
     export PATH=/opt/reg_utils/bin:$PATH
     export PATH=/opt/xhal/bin/:$PATH
 
+    # Add LDQM tools to PATH
+    export PATH=${BUILD_HOME}/gem-light-dqm/dqm-root/bin:$PATH
+    export PATH=${BUILD_HOME}/gem-light-dqm/gemtreewriter/bin:$PATH
```

Comment:
Upon proper setting of env variables, the excutables will be  under {BUILD_HOME}/gem-light-dqm/dqm-root/bin/${XDAQ_OS}/${XDAQ_PLATFORM}

Reply:
I can implement this.  However one suggestion is that the `ldqm` make files allow the packages to be built without requiring these `$ENV` variables.  One should probably make a requirement in the build process that these exist, and terminate if they do not.

Reply:
I think the solution should be slightly different. We will provide a packaging for the lightDQM and the package will be installed somewhere. After this will be done, the script above should simply point to the installation location (or generic env script should include the corresponding folders into the $PATH)

Reply:
Okay in that case I propose to keep the above export's for the time being.

Reply:
(with a subsequent change down the road)

Reply:
well, the rule of thumb now is to compile the code under `gemuser` account where the `env` variables are properly set, so in such case your script won't work! That's why I suggested a change

Reply:
ok, I sign off as is. Will update after packaging is implemented